src/components/TabBar/TabBar.jsx

```diff
@@ -16,7 +16,7 @@ const TabBar = ({ router, tabs, activeTab }) => (
     value={activeTab}
   >
     {tabs.map((tab, index) => (
-      <Tab key={index} className={styles.tab} value={tab.route} label={tab.label} onActive={tab => onActive(tab, router)}>
+      <Tab key={index} className={styles.tab} value={tab.route} label={tab.label.replace(/\//g, '')} onActive={tab => onActive(tab, router)}>
```

Comment:
I assume this is the reason the / was added to all the page names earlier, for some routing reason.